src/main.rs

```diff
@@ -48,6 +48,11 @@ async fn main() -> Result<(), LemmyError> {
   })
   .await??;
 
+  let pool2 = pool.clone();
+  thread::spawn(move || {
+    scheduled_tasks::setup(pool2);
+  });
+
```

Comment:
I messed with things for a while, trying to do it via actix threads, but this seemed to be the only one that creates a non-blocking / background thread.